just-install.json

```diff
@@ -398,6 +398,14 @@
       },
       "version": "5.0.4"
     },
+    "lockhunter": {
+      "installer": {
+        "kind": "innosetup",
+        "x86": "http://lockhunter.com/exe/lockhuntersetup_{{.version}}.exe",
+        "x86_64": "http://lockhunter.com/exe/lockhuntersetup_{{.version}}.exe"
```

Comment:
Lockhunter ships an installer that picks the correct version to install (32 or 64). In that case, should we have entries for both `x86` and `x86_64`?